ranger/gui/ui.py

```diff
@@ -286,10 +284,12 @@ def setup(self):
         from ranger.gui.widgets.taskview import TaskView
         from ranger.gui.widgets.pager import Pager
 
-        # Create a titlebar
+        # Create a titlebar depending whether the setting is "True" or "False"
```

Comment:
Simplify to "Draw titlebar if set." Same for the other comment.

Reply:
Done